config/config.example.yml

```diff
@@ -379,4 +379,10 @@ vocabularies:
 # Default collection/community sorting order at Advanced search, Create/update community and collection when there are not a query. 
 comcolSelectionSort:
   sortField: 'dc.title'
-  sortDirection: 'ASC'
\ No newline at end of file
+  sortDirection: 'ASC'
+
+advancefilter:
```

Comment:
@GauravD2t : A minor suggestion, this should be name `advancedFilters` not `advanceFilter`.  The "d" is important here as it's the "advanced" search feature, not "advance" search.  

I also wonder if we could structure this setting more like this to make it clear it's for "search" only, and make it easier to turn on/off and add filters as a list:
```
# Search settings
search: 
  # Settings to enable/disable or configure advanced search filters.
  advancedFilters:
    enabled: false
    # List of filters to enable in "Advanced Search" dropdown
    filters: [ title, author, subject, entityType ]
```

This would align with @atarix83 's good suggestion to allow this to be enabled or disabled.  It'd also simplify the configuration of filters into an array of values.  (Similar to the "submission > autosave > metadata" config that [already exists](https://github.com/DSpace/dspace-angular/blob/main/src/config/submission-config.interface.ts#L4))

Reply:
channge config